consensus/src/main/java/org/ethereum/beacon/consensus/transition/InitialStateTransition.java

```diff
@@ -51,10 +54,10 @@ public BeaconState apply(BeaconBlock block, BeaconState state) {
 
     ChainStart chainStart = depositContract.getChainStart();
 
-    Builder builder = BeaconState.Builder.createEmpty();
+    MutableBeaconState initialState = MutableBeaconState.createNew();
```

Comment:
What do you think about keeping `BeaconStateImpl.Builder` which is either extends or delegating to `MutableBeaconState`?

Reply:
If extending what should its purpose? 
If delegating then it's another bunch of boilerplate code :/
Also I would like to hide the `BeaconStateImpl` class (of course the Builder can be independent of it)

Reply:
>  If delegating then it's another bunch of boilerplate code :/

Yep, there could be toughs in making changes to `MutableBeaconState` and _not forgetting_ about a `Builder`. Builders are good that they either produce a valid object or throwing an error. But we may keep going without a builder for now.